source/_integrations/recorder.markdown

```diff
@@ -148,6 +148,18 @@ Some installations of MariaDB/MySQL may require an ALTERNATE_PORT (3rd-party hos
 
 <div class='note'>
 
+If using an external MariaDB backend (eg. running on a separate NAS) with home-assistant, you should omit `pymysql` from the url. eg. 
+| Database engine | `db_url`                                                 |
+| :---------------|:---------------------------------------------------------|
+| MariaDB         | `mysql://SERVER_IP/DB_NAME?charset=utf8`                 |
+| MariaDB         | `mysql://user:password@SERVER_IP/DB_NAME?charset=utf8`   |
+
+`pymysql` is not included in the base docker image, and is not necessary for this to work.
+
+</div>
+
+<div class='note'>
```

Comment:
This could be part of the "Custom database engines" table in the same way as PostgreSQL is covered..